packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -42,9 +51,127 @@ The Show-Hide component must be composed inside the [Base][1] component with a l
+    const toggleShowHide = () => setIsOpen(!isOpen);
+
+    const prefix = 'Patients are requesting greater affordability and efficiency in healthcare. With procedures performed in an ambulatory surgery center costing up to 60% less compared to a hospital outpatient department (1), the demand for these facilities is increasing.';
+    const focusableText = 'In fact, the U.S. ambulatory surgery center market is expected to see a 6.9 % compound annual growth rate, reaching $33 billion by 2028. (2) Cerner understands the urgency to grow in the ambulatory surgery center market while continuing to deliver excellent care. Healthcare IT products can help improve clinician efficiency and patient outcomes, as well as enhance communication and data exchange between ambulatory surgery center providers and patients. Read';
+
+    return (
+      <ShowHide focusRef={focusRef} preview={ <p>{prefix}</p> } isOpen={isOpen} onChange={toggleShowHide} >
+        <p>
+          <ShowHideFocuser ref={this.focusRef} prefix={prefix} focusableText={focusableText} />
+        </p>
+      </ShowHide>
+    );
+  }
+  ```
+
+### Multiple children
+  Even though the main purpose of the `ShowHide` component is to truncate long paragraps of texts, the content of the `ShowHide` does not have to be one paragraph. It can include links, lists, and any kind of HTML tags as long as the first line of the hidden text is passed as a string prop to `ShowHideFocuser` component rather than an HTML element.
+  
+Please note:
+  - Links can be placed inside the `p` tag that wraps `ShowHideFocuser` component both preceding and following it. The prop `prefix` can be omitted if content has been passed as an HTML element preceding `ShowHideFocuser`.
+  - Everything that precedes `ShowHideFocuser` component must be also included into the `ShowHide` preview prop.
+  - Do not break lists placing `ShowHideFocuser` inside `li`, as it might cause assistive technologies to read the content incorrectly.
+
+  ```jsx
+    import React, { useState, useRef } from 'react';
+    import ShowHide, { ShowHideFocuser } from 'terra-show-hide';
+
+    const MyComponent = () => {
+      const focusRef = useRef(null);
+      const [isOpen, setIsOpen] = useState(false);
+      const toggleShowHide = () => setIsOpen(!isOpen);
+
+      const header = 'Bringing clarity to life sciences and healthcare decision-making ';
+      const prefix = 'When we accelerate the discovery, development and deployment of life-enhancing insights and therapies, clinicians and patients benefit.';
+      const preview = [
+        <h3>{header}</h3>,
+        <p>{prefix}</p>,
+      ];
+
+      return (
+        <ShowHide focusRef={focusRef} preview={preview} isOpen={isOpen} onChange={toggleShowHide} >
+          <h3>{header}</h3>
+          <p>
+            <ShowHideFocuser ref={this.focusRef} prefix={prefix} focusableText="Several factors help bring clarity to life sciences, researchers and clinicians and " />
```

Comment:
this could use a comment describing exactly what the resulting behavior of this would be.

Reply:
Adde in https://github.com/cerner/terra-core/pull/3838/commits/1813b072e8074cbe8d2ec0d737f42a88b9b81774